doc/classes/EditorExportPlugin.xml

```diff
@@ -158,6 +158,30 @@
 				Return a [PackedStringArray] of additional features this preset, for the given [param platform], should have.
 			</description>
 		</method>
+		<method name="_get_export_option_overrides" qualifiers="virtual const">
+			<return type="Dictionary" />
+			<param index="0" name="platform" type="EditorExportPlatform" />
+			<description>
+				Return a [Dictionary] of override values for export options, that will be used instead of user-provided values. Overridden options will be hidden from the user interface.
+				[codeblock]
+				class MyExportPlugin extends EditorExportPlugin:
+				    func _get_name() -&gt; String:
+				        return "MyExportPlugin"
+
+				    func _supports_platform(p_platform) -&gt; bool:
+				        if p_platform is EditorExportPlatformPC:
+				            # Run on all desktop platforms including Windows, MacOS and Linux.
+				            return true
+				        return false
+
+				    func _get_export_options_overrides(p_platform) -&gt; Dictionary:
+				        # Override "Embed PCK" to always be enabled.
+				        return {
+				            'binary_format/embed_pck': true,
+				        }
```

Comment:
Minor nitpicks as per GDScript conventions.
```suggestion
				    func _supports_platform(platform) -&gt; bool:
				        if platform is EditorExportPlatformPC:
				            # Run on all desktop platforms including Windows, MacOS and Linux.
				            return true
				        return false

				    func _get_export_options_overrides(platform) -&gt; Dictionary:
				        # Override "Embed PCK" to always be enabled.
				        return {
				            "binary_format/embed_pck": true,
				        }
```

Reply:
Thanks! Fixed in my latest push :-)